api/src/main/java/care/smith/fts/api/cda/BundleSender.java

```diff
@@ -3,16 +3,17 @@
 import care.smith.fts.api.TransferProcessStep;
 import care.smith.fts.api.TransferProcessStepFactory;
 import care.smith.fts.api.TransportBundle;
+import care.smith.fts.api.rda.BundleSender.Result;
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
 public interface BundleSender extends TransferProcessStep {
 
-  Mono<Result> send(Flux<TransportBundle> bundles);
+  Mono<Result> send(TransportBundle bundle);
 
   interface Factory<C> extends TransferProcessStepFactory<BundleSender, Config, C> {}
 
   record Config() {}
 
-  record Result(int bundleCount) {}
+  record Result() {}
```

Comment:
Do we really need an empty result?

Reply:
We need at least something to count. And as I didn't want to use String here, and Mono<Void> would result in an empty result which could not be counted, an empty record was okay for me.